### ISSUE TITLE
Implement a tool for restoring unsaved projects 

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#7956]
### Description

Implement Unsaved Project Recovery

[Designs](https://www.figma.com/design/4NexYrBLVql4nMJfkwLXDc/40---Crash-recovery?node-id=6431-41979&t=xlWIJuVxzad3nBRO-4) 

AU3 bugs which may be fixed along the way:

- [x] https://github.com/audacity/audacity/issues/7937
- [x] https://github.com/audacity/audacity/issues/3163
- [ ] https://github.com/audacity/audacity/issues/3566
- [ ] https://github.com/audacity/audacity/issues/1485
- [x] https://github.com/audacity/audacity/issues/7866

**Comment by Tantacrul** [comment]
@chinakov And @dozzzzer - how are we doing with project recovery? Is it working in general (excluding these issues from AU3)?

**Comment by dozzzzer** [comment]
@Tantacrul Yes, it restores the project after a crash or discards unsaved changes, depending on your choice.

<img width="308" height="158" alt="Image" src="https://github.com/user-attachments/assets/bbf292fb-aa75-474b-b7a8-df115eef37f8" />

**Comment by kryksyh** [comment]
Estimation: 3 days AU3 + 3 days AU4

**Comment by DilsonsPickles** [comment]
@Tantacrul @teetow  is this guy coming with us to AU4?

<img width="508" height="340" alt="Image" src="https://github.com/user-attachments/assets/3b7aa4b1-0b67-43c4-868c-9f0cab8a0da6" />

**Comment by Tantacrul** [comment]
We should probably match this exactly, unless we've a firm idea about what the alternative should be.

One obvious alternative is to not have this be so complicated. We could limit recovery to just one project (the most recent) and provide a simple modal that tells says 

> "Audacity has detected a project that was not saved properly. Would you like to recover it?"

> Project: [description of project]

> [Discard project] [Recover project] [Skip}


@kryksyh - do you have a preference here from a technical POV?

@teetow @dozzzzer  - any thoughts?

**Comment by dozzzzer** [comment]
@Tantacrul 

> We could limit recovery to just one project (the most recent) and provide a simple modal 

I think that's our best bet.

**Comment by Tantacrul** [comment]
> I think that's our best bet.

Cool. Assuming no technical issues, we should do that.

**Comment by DilsonsPickles** [comment]
[Link to Figma designs](https://www.figma.com/design/4NexYrBLVql4nMJfkwLXDc/40---Crash-recovery?node-id=6431-41979&t=xlWIJuVxzad3nBRO-4)

UX flow remains the same as AU3. Link is to updated table design.

<img width="1400" height="920" alt="Image" src="https://github.com/user-attachments/assets/f8878392-0956-47db-9b39-c4c1f62e8abd" />

**Comment by kryksyh** [comment]
I updated the title, since this task is about recovering an unsaved project.

**Comment by DilsonsPickles** [comment]
@kryksyh do we have all the relevant UI designed now?

**Comment by DilsonsPickles** [comment]
Working [Figma file](https://www.figma.com/design/4NexYrBLVql4nMJfkwLXDc/40---Project-recovery?node-id=10379-6289&t=IgMVL0W1E0Zjx1xk-4)

@kryksyh I'm still making tweaks for this. Please confirm this is the correct work for this ticket.

**Comment by DilsonsPickles** [comment]
Just need to finish styling slides and then this is gtg
